shvutil.c

```diff
@@ -138,7 +138,7 @@ ShUtilvAllocateContiguousMemory(
 	// If we're here then we'll just fall back to using executable
 	// memory.
 	//
-#pragma warning(suppress: 30029)
+#pragma warning(suppress: 30030)
```

Comment:
There is a warning/suppress.h which has #define names for all of these, to make it clearer. You should also use the 'comment' feature of the supression to indicate why it's being supressed.